diff --git a/src/utils/updater.ts b/src/utils/updater.ts
index f99c6ca1..d432f988 100644
--- a/src/utils/updater.ts
+++ b/src/utils/updater.ts
@@ -38,6 +38,8 @@ async function Unwrap<T>(p: Promise<IpcRes<T>>) {
 }
 
 export async function checkForUpdates() {
+    return (isOutdated = false);
+
     changes = await Unwrap(VencordNative.updater.getUpdates());
     if (changes.some(c => c.hash === gitHash)) {
         isNewer = true;
@@ -63,6 +65,8 @@ export async function update() {
 export const getRepo = () => Unwrap(VencordNative.updater.getRepo());
 
 export async function maybePromptToUpdate(confirmMessage: string, checkForDev = false) {
+    return;
+
     if (IS_WEB || IS_UPDATER_DISABLED) return;
     if (checkForDev && IS_DEV) return;
 
