diff --git a/scripts/build/common.mjs b/scripts/build/common.mjs
index e88f1e2b..3629cd6e 100644
--- a/scripts/build/common.mjs
+++ b/scripts/build/common.mjs
@@ -42,7 +42,7 @@ export const IS_DEV = watch || process.argv.includes("--dev");
 export const IS_REPORTER = process.argv.includes("--reporter");
 export const IS_STANDALONE = process.argv.includes("--standalone");
 
-export const IS_UPDATER_DISABLED = process.argv.includes("--disable-updater");
+export const IS_UPDATER_DISABLED = true;
 export const gitHash = process.env.VENCORD_HASH || execSync("git rev-parse --short HEAD", { encoding: "utf-8" }).trim();
 
 export const banner = {
